lib/errorTypes.js

```diff
@@ -2,5 +2,8 @@
 
 var errors = require('restify-errors');
 
-errors.makeConstructor('RequestCloseError');
-errors.makeConstructor('RouteMissingError');
+// This allows Restify to work with restify-errors v6+
```

Comment:
Just out of curiosity: what is the change in restify v6+ that requires this?

Reply:
In v5 of `restify-errors` the function call to `makeConstructor` added the constructor to `restify-errors` exports. This changed in v6 and the recommendation was that the consumer should keep a reference to the new error constructor.